docs/fides/docs/config/index.md

```diff
@@ -0,0 +1,103 @@
+# Configuration
+
+---
+
+## Setting Configuration Values
+
+Fides can be configured in two different ways: either via a `toml` file or via environment variables.
+Both methods can be used simultaneously, with environment variables taking precedence over the `toml` file values.
+
+### Using a Configuration File
+
+Fides will use the first config file it can read from the following locations. Listed in order of precedence they are:
+
+1. At the path specified using the config file argument passed through the CLI, i.e. `fides -f <config_path>`
+1. At the path specified by the `FIDES__CONFIG_PATH` environment variable
+1. In the current working directory it will check for a subdir `.fides` and a file within named `fides.toml`, i.e. `./.fides/fides.toml`
+
+### Generating a Config File
+
+If you'd like to generate a new config file automatically using default values, run `fides init`.
+This will create the file at the default location of `./.fides/fides.toml`
+
+### Setting Values Via Environment Variables
+
+Fides follows a set pattern for configuration via environment variables.
+It looks for environment variables that start with `FIDES` followed by the config subsection name, followed by the key name, all separated with double underscores.
+In practice this would look like `FIDES__<SUBSECTION>__<KEY>`
+
+As a `toml` configuration value:
+
+```toml
+[database]
+host = config_example
+```
+
+As an environment variable:
+
+```sh
+EXPORT FIDES__DATABASE__HOST=config_example
+```
+
+---
+
+## Viewing your configuration
+
+You can view the current configuration of your application via either the CLI or API.
+
+### CLI
+
+To view your application configuration via the CLI, run:
+
+```sh
+fides view config
+```
+
+This will show _all_ configuration variables, including sensitive ones.
```

Comment:
This is out of scope of this PR, but would it be possible to make this not log sensitive values by default and add a opt it option to out the sensitive values. Something like `fides view config --show-sensitive-values`. I'm sure at some point someone is going to just run `fides view config` blindly and be surprised that it logged the sensitive values.

Reply:
Do you have concerns around logging sensitive info to a CLI user? This is a CLI-specific command, so any values that get logged they would have access to directly via their local config file or env vars anyway I think

Reply:
That's true. Since they already have access locally it seems fine.

It's less about if it's secure and more about what the user expects. If CLI users are expecting this command to log everything then I'm on board

Reply:
I don't know why they wouldn't to be honest, but maybe? I can add a note to that effect in another PR I have open to update the CLI docstrings